### DISCUSSION TITLE
is there a data viewer for EdgeDB?

### DISCUSSION THREAD

**Discussion by tadasgedgaudas** [geldata/gel#2544] [General]
Had a chance to play with EdgeDB and I love the database.

But I cannot find a data viewer for EdgeDB like DBeaver or Datagrip?

Does anyone know if such thing exists?

**Comment by 1st1** [comment]
> Had a chance to play with EdgeDB and I love the database.

Thank you for the great feedback!

> But I cannot find a data viewer for EdgeDB like DBeaver or Datagrip?

Yes, a GUI is in the works. There's no concrete ETA, but we hope to have a preview of it in 1 or 2 months.

**Reply by knaitas** [reply]
Thanks for the answwer.

**Reply by agritheory** [reply]
> > Had a chance to play with EdgeDB and I love the database.
> 
> Thank you for the great feedback!
> 
> > But I cannot find a data viewer for EdgeDB like DBeaver or Datagrip?
> 
> Yes, a GUI is in the works. There's no concrete ETA, but we hope to have a preview of it in 1 or 2 months.

I _assume_ it is web-based? What can you share at this point?

**Reply by 1st1** [reply]
It will be web-based (just open the local DB instance URL in a browser) as well as an electron app. It's still in the making, we're redoing the UI. We'll put it out as soon as we can.

**Comment by bernardro** [comment]
Hi,
Any update on this?
Willing to help with testing

**Reply by derekjwilliams** [reply]
Looks like this is what is available: https://www.edgedb.com/blog/edgedb-2-0#edgedb-ui
